storage-proofs/src/hasher/pedersen.rs

```diff
@@ -432,4 +581,42 @@ mod tests {
 
         assert_eq!(val, val_back);
     }
+
+    // Asserts that hashing 118 bytes using `sapling`'s Pedersen Hash yields the same digest as
+    // hashing with `PedersenHasher`. 118 bytes is the largest buffer that `sapling`'s Pedersen Hash
+    // can handle before `panic`ing (`sapling` only generates 5 segments worth of generators, so it
+    // will run out of generators if hashing more than 118 bytes).
+    #[test]
```

Comment:
It would be worth spelling out a little more clearly how the 118 is derived — so readers not intimately familiar with this pedersen hashing can follow/verify this logic.

Each segment contains 63 3-bit chunks, so 5 segments support 3 * 5 * 63 = 945 bits. So 118 bytes = 118 * 8 = 944 bits, making 118 the larger number of whole bytes that can be handled.

Although this is implicit, writing it out will be helpful for those coming up to speed in the future.